Gemfile

```diff
@@ -20,3 +22,11 @@ group :development, :test do
   gem 'better_errors'
   gem 'binding_of_caller'
 end
+
+group :test do
+  gem "faker", "~> 1.4.3"
```

Comment:
I've kind of found faker to be an anti-pattern. It's not bad per se, but when I used it ended up trying to make _everything_ use fake data and it slowed me down a lot. Keep it in for now and you may find that you use it just fine.